### PR TITLE
FINERACT-1528: Replace copy and pasted Swagger UI with Springdoc

### PR DESCRIPTION
**Pull request by vidakovic**
## Description

Introduce Springdoc instead manual Swagger UI. Now the Swagger test UI can be turned off in production. Moved legacy doc into a separate folder; now that we have the new documentation online we could consider getting rid of it.

### CODE REVIEW

File: fineract-client/build.gradle [hunk 1]
```diff
@@ -72,26 +72,23 @@ task buildTypescriptAngularSdk(type: org.openapitools.generator.gradle.plugin.ta
...
-        npmName: 'apache-fineract-client'
+        npmName: '@apache/fineract-client',
+        npmRepository: "${npmRepository}"
     ]
     dependsOn = [
         ':fineract-provider:resolve'
     ]
 }
 
-licenseFormatMain.dependsOn buildJavaSdk
```
Comment (galovics) [inline fineract-client/build.gradle]: Wait a sec. Are you sure this is gonna be replaced by the change below?

When I was fixing the incremental build for Gradle, this combination worker flawlessly. Have you tested whether it's broken?

**Comment by vidakovic** [comment]
yeah... that was an accident...

**Comment by vidakovic** [comment]
@galovics ... really don't know why that specific test is breaking... didn't touch anything concerning it. Any ideas? Is this happening in develop too?

File: fineract-client/build.gradle [hunk 2]
```diff
@@ -72,21 +72,21 @@ task buildTypescriptAngularSdk(type: org.openapitools.generator.gradle.plugin.ta
...
```
Comment (galovics) [inline fineract-client/build.gradle]: @vidakovic are you sure this is not gonna be needed?

**Comment by vidakovic** [comment]
@galovics great... it worked... but the tests are a bit wonky at the moment.

**Review by galovics — APPROVED** [review]

**Pull request merged**